fvtest/compilertest/build/toolcfg/gnu/common.mk

```diff
@@ -26,18 +26,27 @@ SED_PATH?=sed
 AR_PATH?=ar
 PERL_PATH?=perl
 
+ifeq ($(OS),osx)
```

Comment:
This seems unintuitive enough that it deserves at least a comment to explain. "as" works for me on OSX. Is there some version of OSX/clang where "clang" is the right way to access the assembler (or have I misunderstood) ?